doc/word.xml

```diff
@@ -0,0 +1,104 @@
+#############################################################################
+##
+#W  word.xml
+#Y  Copyright (C) 2020                                      Maria Tsalakou
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+
+<#GAPDoc Label="WordToString" >
+<ManSection>
+  <Oper Name = "WordToString" Arg = "alphabet, word"
+    Label="for a string and a list"/>
+  <Returns>
+    The <A>word</A> in the form of a string.
+  </Returns>
+  <Description>
+    Returns the word <A>word</A> that is given in the form of a list, in 
+    the form of a string of letters of the <A>alphabet</A>.
+ <Example><![CDATA[
+gap> WordToString("abcd", [4, 2, 3, 1, 1, 4, 2, 3]);
+"dbcaadbc"]]></Example>
+  </Description>
+ </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="RandomWord" >
+<ManSection>
+  <Oper Name = "RandomWord" Arg = "length, number_letters"
+    Label="for two integers"/>
+  <Returns>
+    A random word of length <A>length</A> in the form of a list using 
+    <A>number_letters</A> letters.
+  </Returns>
+  <Description>
+	  Returns a random word of length <A>length</A>
```

Comment:
This line effectively repeats the contents of `<Returns>`. Maybe you could either move some of the stuff you wrote  in the "returns" section to the description section, or you could add some additional comments in the description regarding how the random word is generated, just to avoid repetition?